src/pudl/helpers.py

```diff
@@ -1476,6 +1476,28 @@ def dedupe_and_drop_nas(
     )
 
 
+def drop_records_with_null_pk(
+    df: pd.DataFrame, primary_key_col: str, num_of_expected_nulls: int
```

Comment:
As a more general helper function, would it make sense for the PK to be a list of column names instead, to allow for composite primary keys?

Reply:
i can imagine that being helpful but for the sake of time on this project i will generalize this a little bit to just be nulls in any column and add a note that this would be good to later generalize.